core/src/main/java/io/questdb/griffin/engine/groupby/vect/MaxTimestampVectorAggregateFunction.java

```diff
@@ -115,4 +115,9 @@ public void clear() {
     public long getTimestamp(Record rec) {
         return max.longValue();
     }
+
+    @Override
+    public boolean isReadoutStateless() {
+        return false;
```

Comment:
Should not be different from `MaxLongVectorAggregateFunction`

Reply:
should be `false` for all aggregation functions